screens/AccountCloseConfirmForm.js

```diff
@@ -0,0 +1,452 @@
+                					</View>
+												:
+													Object.keys(mostrecentShift.close.closingBalances).length == 0 
+													&&
+														null
+                	}                	
+              	</View>       
+
+            	</View>             		
+          	</View>
+        	{/* #Post-Closing Entries */}
+
+					{/* Cash */}
+          	<View style={styles.sectionBar}>             
+            	<View style={[styles.tableCellView, {flex: 4}]}>
+              	<TouchableOpacity>
+                	<Text style={styles.sectionBarTextSmall}>
+                  	{t('cash')}
+                	</Text>
+              	</TouchableOpacity>
+            	</View>
+          	</View>
+
+          	<View style={[styles.container, styles.no_mgrTop]}>
+            	<View style={{flex: 3, justifyContent: 'center'}}>
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Starting Cash 
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;{mostrecentShift.open.balance}</Text>
+                	</View>
+              	</View>
+
+              	{
+									mostrecentShift.close.closingShiftReport !== null && 
+									mostrecentShift.close.closingShiftReport.totalByPaymentMethod !== null && 
+									mostrecentShift.close.closingShiftReport.totalByPaymentMethod.hasOwnProperty('CASH') &&  (
+									<View style={styles.fieldContainer}>
+                		<View style={{flex: 3}}>
+                  		<Text style={[styles.fieldTitle]}>
+                  		Total Sales
+                  		</Text>
+                		</View>
+                		<View style={{flex: 1}}>
+                  		<Text>
+                  			$&nbsp;{                  				
+                  				mostrecentShift.close.closingShiftReport.totalByPaymentMethod.CASH.orderTotal
+												}
+                  		</Text>
+                			</View>
+              		</View>
+              	)}                        
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                  	Amount of payment
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Text>$&nbsp;?</Text>
+                	</View>
+              	</View>     
+
+              	<View style={styles.fieldContainer}>
+                	<View style={{flex: 3}}>
+                  	<Text style={[styles.fieldTitle]}>
+                    	Amount of cash register should have
+                  	</Text>
+                	</View>
+                	<View style={{flex: 1}}>
+                  	<Field
```

Comment:
This shouldn't be a text field. It should display the amount of (starting cash + total cash sales).

Reply:
mostrecentShift.open.balance + mostrecentShift.close.closingShiftReport.totalByPaymentMethod.CASH.orderTotal